packaging/nnstreamer.spec

```diff
@@ -327,6 +341,18 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %{_libdir}/pkgconfig/capi-nnstreamer.pc
 %{_libdir}/libcapi-nnstreamer.so
 %{_libdir}/libcapi-nnstreamer.a
+
+%files -n capi-nnstreamer-single-new
+%manifest capi-nnstreamer.manifest
+%license LICENSE
+%{_libdir}/libcapi-nnstreamer-single-new.so.*
+
+%files -n capi-nnstreamer-single-new-devel
+%{_includedir}/nnstreamer/nnstreamer.h
+%{_includedir}/nnstreamer/nnstreamer-single.h
+%{_libdir}/pkgconfig/capi-nnstreamer.pc
```

Comment:
You can't test this library with this .pc file. (it won't direct libcapi-nnstreamer-single-new.so)

Reply:
updated.